worlds/metroidprime/Locations.py

```diff
@@ -0,0 +1,237 @@
+from typing import Dict, List, Tuple
+from BaseClasses import Location
+
+from .MetroidPrimeInterface import MetroidPrimeLevel
+
+METROID_PRIME_LOCATION_BASE = 5031100
+
+
+class MetroidPrimeLocation(Location):
+    game: str = "Metroid Prime"
+
+
+chozo_location_table = {
+    "Chozo Ruins: Main Plaza - Half-Pipe": 5031100,
+    "Chozo Ruins: Main Plaza - Grapple Ledge": 5031101,
+    "Chozo Ruins: Main Plaza - Tree": 5031102,
+    "Chozo Ruins: Main Plaza - Locked Door": 5031103,
+    "Chozo Ruins: Ruined Fountain": 5031104,
+    "Chozo Ruins: Ruined Shrine - Plated Beetle": 5031105,
+    "Chozo Ruins: Ruined Shrine - Half-Pipe": 5031106,
+    "Chozo Ruins: Ruined Shrine - Lower Tunnel": 5031107,
+    "Chozo Ruins: Vault": 5031108,
+    "Chozo Ruins: Training Chamber": 5031109,
+    "Chozo Ruins: Ruined Nursery": 5031110,
+    "Chozo Ruins: Training Chamber Access": 5031111,
+    "Chozo Ruins: Magma Pool": 5031112,
+    "Chozo Ruins: Tower of Light": 5031113,
+    "Chozo Ruins: Tower Chamber": 5031114,
+    "Chozo Ruins: Ruined Gallery - Missile Wall": 5031115,
+    "Chozo Ruins: Ruined Gallery - Tunnel": 5031116,
+    "Chozo Ruins: Transport Access North": 5031117,
+    "Chozo Ruins: Gathering Hall": 5031118,
+    "Chozo Ruins: Hive Totem": 5031119,
+    "Chozo Ruins: Sunchamber - Flaaghra": 5031120,

```

Comment:
I should have brought this up sooner given I'm making the tracker, but Flaahgra's name is misspelled here and in a few other places.

Reply:
I'll add changing these to my PR